### PR TITLE
Initial CI setup

### PR DESCRIPTION
**Pull request by TSC21**
This is an initial setup for CI in the Yukon package and pretends to answer the following:

1. Setup CI for testing and deployment
- Ensure unit tests are run as part of the build and when PRs are submitted.
- Use our Buildkite organization and workers

2. Setup auto deployment of the Yukon python package to Pypi

*Toxic* is still not in place but should be done soon as well. *sonarqube* will follow it.

Also did some cleanup while moving the back and frontend code to the *src* folder.

### CODE REVIEW

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+          propagate-environment: true
+          environment:
+            - "SONARQUBE_TOKEN"
+            - "PYPI_PASSWORD"
+            - "BUILDKITE"
+            - "BUILDKITE_BUILD_NUMBER"
+    agents:
+      queue: 'default'
+    timeout_in_minutes: 15
+  # TODO: add verification builds
```
Comment (thirtytwobits) [inline .buildkite/pipeline.yml]: You should add the verification builds in the Buildkite UI itself (see https://buildkite.com/uavcan/nunavut-pr/settings). The reason is any third party can initiate a verification build by submitting a PR to our organization's repo. If you pull the verification build steps from source then the committer can easily steal secrets and take over your CI possibly gaining access to your source repositories.

So, remove this TODO from here.

File: LICENSE.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 UAVCAN
+Copyright (c) 2020 Amazon
```
Comment (thirtytwobits) [inline LICENSE.md]: Please use your own copyright (Incessant Gravity, right?). You are writing this code and releasing it under MIT. Amazon is simply supporting your development efforts within the terms of our contract. You'll see we specify the copyright in that contract is to be your own.

Comment (TSC21) [reply LICENSE.md]: I kept the previous Copyright and added my own.

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (TSC21) [reply .buildkite/pipeline.yml]: Removed

File: .buildkite/pipeline.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      - ".tox/flake8/tmp/*"
+      - ".tox/dist/*.zip"
+      - ".tox/py*-test/tmp/xunit-result.xml"
+    plugins:
+      - docker#v3.3.0:
+          workdir: /repo
+          image: "uavcan/toxic:py35-py38-sq"
+          propagate-environment: true
+          environment:
+            - "SONARQUBE_TOKEN"
```
Comment (thirtytwobits) [inline .buildkite/pipeline.yml]: I'll have to upload these secrets for you. I'll do that now.

Comment (TSC21) [reply .buildkite/pipeline.yml]: Thanks!

**Pull request merged**